contracts/wrapper/NameWrapper.sol

```diff
@@ -1096,4 +1181,12 @@ contract NameWrapper is
             ownerOf(uint256(node)) != address(0) &&
             ens.owner(node) == address(this);
     }
+
+    function _isNotDotEthOrIsNotExpired(
```

Comment:
I don't follow the logic here. Why do we restrict modifying the subname to non .eth 2LDs that aren't expired? And why do we apply the grace period to non .eth 2LDs? Don't we only have the grace period on .eth 2LDs?

Reply:
This is something you yourself put in when we added the grace period to expiry. If it's not a .eth, we just ignore and return true. We don't apply grace period to non .eth 2LDs, if it's not a .eth the statement just returns true.